events/README.md

```diff
@@ -0,0 +1,472 @@
+- Query
+  - None
+- Body
+  - None
+- Headers
+  - Content-Type: application/json
+  - Authorization: Bearer <management_token>
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```jsx
+      {
+      	"limit": 10,
+      	"data": [
+      		<event_object>,
+      		<event_object>,
+      	]
+      }
+      ```
+- **Error Response:**
+  - **Code:** 401
+    - **Content:**
+      `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
+  - **Code:** 500
+    - **Content**
+      `{ 'statusCode': 500, 'error': 'Internal server error', 'message': 'Couldn't get events. Please try again later'`
+
+## POST - /events/join
+
+Generate an auth token for a peer to join a event.
+
+To join the event (for guests, host, maven, moderator)
+
+- **Params:**
+  - None
+- **Query**
+  - None
+- **Body**
+  - Required: **`event_id=[string]`** (The ID of the event to which the user belongs)
+  - Required: **`user_id=[string]`** (The ID of the user for whom the token is being generated)
+  - Required: **`role=[string]`** (The role of the user in the event)
+  ```json
+  {
+    "event_id": "<event_id>",
+    "user_id": "<user_id>",
+    "role": "<role>"
+  }
+  ```
+- **Headers**
+  - Content-Type: application/json
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```json
+      {
+        "token": "<auth_token>",
+        "msg": "Token generated successfully!",
+        "success": true
+      }
+      ```
+- **Error Response:**
+  - **Code:** 500
+    - **Content**
+      **`{ "msg": "Some error occured!", "success": false }`**
+
+## GET - /events/:id=<event_id>
+
+Retrieves the details of a specific active event_id.
+
+- **Params:**
+  - **`id=[string]`** (The ID of the event to retrieve)
+- **Query**
+  - None
+- **Body**
+  - None
+- **Headers**
+  - Authorization: Bearer <management_token>
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```json
+      {
+        "id": "<event_id>",
+        "name": "<event_name>",
+        "customer_id": "<customer_id>",
+        "session": {
+          "id": "<session_id>",
+          "created_at": "YYYY-MM-DDTHH:MM:SS.sssZ",
+          "peers": [
+            <peer_object>,
+            <peer_object>,
+            <peer_object>
+          ]
+        }
+      }
+      ```
+- **Error Response:**
+  - **Code:** 401
+    - **Content:**
+      **`{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`**
+  - **Code:** 404
+    - **Content**
+      **`{ 'statusCode': 404, 'error': 'Not Found', 'message': 'Event not found' }`**
+  - **Code:** 500
+    - **Content**
+      **`{ 'statusCode': 500, 'error': 'Internal server error', 'message': 'Unable to retrieve event details' }`**
+
+## PUT - /events
+
+Update the event, make event enabled/disabled,
+
```

Comment:
I too think we should use PATCH here. 
@satyam73 please share your thoughts. 

We had discussed to use PATCH for delete since we were not actually deleting anything.